resume.md

```diff
@@ -1,49 +1,65 @@
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
+Academic Intern: UC Berkeley Computer Science Department | Berkeley, CA | June 2019 - Dec 2019
+* Assisted in labs, office hours, and coding assignments for students in CS61A and CS61B
+* Contributed 100+ hours over 2 semesters
+* Beta tested and gave feedback on a newly designed class project for CS61A about building a spellchecker system
+
+Front End Developer: Pioneers in Engineering Club | Berkeley, CA | Aug 2018 - May 2019
+* Helped build interactive web apps and tools in HTML/CSS/JavaScript as part of PiE’s annual high school robotics competition
+* Contributed 120+ hours in work sessions and game days; tools included visual display of the scoreboard and timer, forms for inputting team and round information, and GUI that players use to make game-specific decisions during the round
+
+Medical Device Intern: Children’s Hospital Oakland | Oakland, CA | June 2018 – Aug 2018
+* Measured hemoglobin levels of Sickle Cell patients using the HemeChip™ device
+* Discovered severe discrepancies in HemeChip™ results compared to trusted hemoglobin measurement assays; presented findings in a scientific poster at research symposium
+
+Galaxy Explorer Program Intern: Chabot Space & Science Center | Oakland, CA | June 2015 – June 2018
+* Demoed hands-on science experiments to museum visitors
+* Supervised volunteer science demonstrators; served as the communication link between volunteers and museum staff
 
-**Software Engineer Intern**: Company | City, State ... *Month 20XX - Month 20XX*
-* Conducted A/B Test using FRAMEWORK to test EXPERIENCE in order to increase METRIC resulting in NUMBER. 
-* Utilized DEBUG TECHNOLOGY to resolve recurring crashes (X% of sessions) in beta before hitting production..
-* ...
 
 ## PROJECTS
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Developed PROJECT NAME in order to FEATURE solving PROBLEM for POPULATION
-* Launched PROJECT NAME to the Apple App Store resulting in X Downloads and Y Monthly Users
-* Beta Tested to X Number of users and collected Crash Data using analytics LIBRARY
-* ...
-
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Contributed to OPEN SOURCE LIBRARY, a popular module for FEATURE, by closing X tasks and merging Y Pull Requests.
-* Created Demo Tutorial series to demonstrate FEATURE for early adopters prior to production push.
-* Managed App Version release cycles to maintain proper software versioning for major and minor app increments.
-* ...
+Quadcopter Drone Assembly: Built and programmed DIY quadcopter | Chabot Space & Science Center
+* Led a team of 5 students in building a quadcopter by researching and procuring the components and manually assembling them
+* Programmed an Arduino flight controller in C++; soldered connections between flight controller, motors, and remote control receiver
```

Comment:
what were the technical challenges you had to overcome?  Did you architect the project?  Any interesting design patterns?